eva_submission/eload_submission.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, eload_number: int):
         self.eload = f'ELOAD_{eload_number}'
         self.eload_dir = os.path.abspath(os.path.join(cfg['eloads_dir'], self.eload))
         self.eload_cfg = EloadConfig(os.path.join(self.eload_dir, '.' + self.eload + '_config.yml'))
+        self.maven_env = cfg['maven']['environment']
+        self.maven_settings_file = cfg['maven']['settings_file']
```

Comment:
These are only used by the `get_metadata_connection_handle` 
It might be slightly more elegant to have a `@property` in `Eload` to get connections.